bundles/test/org.eclipse.smarthome.magic/web/index.html

```diff
@@ -0,0 +1,23 @@
+<html>
```

Comment:
Could we add some love to that page?
This now looks like a 90ies webpage prototype in Netscape, this is not necessarily a good way to make newbies fall in love with the project...

Reply:
Done